test/integration/interrupt_resume_test.rb

```diff
@@ -89,19 +89,19 @@ def test_interrupt_resume_when_table_has_completed
       ghostferry.send_signal("TERM")
     end
 
-    dumped_state = ghostferry.run_expecting_interrupt
-    assert_basic_fields_exist_in_dumped_state(dumped_state)
+    ghostferry.run
 
-    # Resume ghostferry from interrupted state
-    datawriter = new_source_datawriter
-    ghostferry = new_ghostferry(MINIMAL_GHOSTFERRY)
+    assert_test_table_is_identical
 
-    start_datawriter_with_ghostferry(datawriter, ghostferry)
-    stop_datawriter_during_cutover(datawriter, ghostferry)
+    assert ghostferry.logrus_lines["ferry"].length > 0
 
-    ghostferry.run(dumped_state)
+    ghostferry.logrus_lines["ferry"].each do |line|
+      if line["msg"].start_with?("Received signal: ")
+        assert line["msg"].match?("Received signal: terminated during cutover. " \
```

Comment:
and now we need an assertion that the assertion ran 😛 

(because there's no guarantee that any of the array elements `start_with?("Received signal")`)